dockers/llm.rag.service/common.py

```diff
@@ -373,4 +396,55 @@ def convert_sql_result_to_nl(state: State, model_id, llm):
     response = llm.invoke(trimmed_prompt)
     logger.info(f"LLM generated NL answer to user question: {response.content}")
 
-    return {"answer": response.content}
\ No newline at end of file
+    return {"answer": response.content}
+
+
+
+def get_answer_with_settings_with_weaviate_filter(question, vectorstore, client, model_id, max_tokens, model_temperature, system_prompt, relevant_docs):
```

Comment:
I think that in order to connect sql search with vector search we can puch the "question selection" here.